lib/main/atom/onSaveHandler.ts

```diff
@@ -6,7 +6,8 @@ import atomConfig = require('./atomConfig'); ///ts:import:generated
 ///ts:import=parent
 import parent = require('../../worker/parent'); ///ts:import:generated
 
-import {errorView, show} from "./views/mainPanelView";
+import {getFileStatus} from "../atomts";
```

Comment:
> Shouldn't it have crashed Atom or something?

Nope. Its because we only use items *off* of the export *from functions*. This means that when these functions *actually execute* the are defined. If we used these items at the root level they would have been `undefined`. 

More: https://nodejs.org/api/modules.html#modules_cycles